server/sonar-web/src/main/js/store/appState/duck.ts

```diff
@@ -17,62 +17,52 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-// @flow
-/*::
-type AppState = {
-  adminPages?: Array<*>,
-  authenticationError: boolean,
-  authorizationError: boolean,
-  organizationsEnabled: boolean,
-  qualifiers: ?Array<string>
-};
-*/
+interface AppState {
```

Comment:
I haven't changed this file in the end, but I decided to not throw my work away 😄